docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -465,9 +471,37 @@ By default, string-based permission is performed by the `io.quarkus.security.Str
 <5> Permissions are not beans, therefore only way to obtain bean instances is programmatically via the `Arc.container()`.
 
 CAUTION: If you plan to use the `@PermissionsAllowed` on the IO thread, review the information in xref:security-proactive-authentication-concept.adoc[Proactive Authentication].
+
 NOTE: The `@PermissionsAllowed` is not repeatable on class-level due to limitations of Quarkus interceptors.
 Please find well-argued explanation in the xref:cdi-reference.adoc#repeatable-interceptor-bindings[Repeatable interceptor bindings] section of the CDI reference.
 
+Provided `SecurityIdentity` contains roles, the easiest to add permissions to `SecurityIdentity` is to create a roles to permissions mapping.
+<<authorization-using-configuration,HTTP role-based policies>> can be used to grant `SecurityIdentity` permissions required by `CRUDResource` endpoints to authenticated requests like this:
+
+[source,properties]
+----
+quarkus.http.auth.policy.role-policy1.roles-allowed=**                                              <1>
```

Comment:
I don't think it is necessary to highlight that both the roles and permissions access control can be enabled as bullet number 3 makes it clear this is part of HTTP roles policy and that you need to be authenticated to use it.

TBH it is not very useful, because if you secure path for roles, you already know this role has this permission, therefore specifying roles allowed only makes sense in some edge case where you want to secure path for 3 roles and grant permissions to only one of them. It's simpler without `roles-allowed` line, I agree.